specification/immunisation-fhir-api.yaml

```diff
@@ -444,8 +444,8 @@ components:
                 type: string
                 description: |
                   The vaccine preventable disease the dose is being administered against.
-                  Expressed as `<value>` where`<value>` must be a valid Snomed Code.
-                example: "39114911000001105"
+                  Expressed as `<value>` where`<value>` must be a valid Disease Type (COVID19, FLU, HPV, MMR).
```

Comment:
?Should be vaccine type, not disease type

Reply:
as discussed, this is because the query parameter is diseaseType